.travis.yml

```diff
@@ -18,6 +18,10 @@ install:
             mkdir -p ~/maven/$MAVEN_VERSION && \
             wget -O - http://www.eu.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | \
                 tar -C ~/maven/$MAVEN_VERSION --strip-components=1 -xzf -; }
+    - sudo apt-get update && sudo apt-get -y install python-pip
+    - sudo pip install Jinja2
+    - sudo pip install MySQL-python
+    - sudo pip install requests
```

Comment:
Maybe move the dependencies to a `requirements.txt` and set version numbers explicitly: https://github.com/inodb/revmut/blob/master/requirements.txt. Then install with: `pip install -r requirements.txt`. I doubt any of these packages will have many backwards incompatible changes, but it's good to be explicit anyway.